packages/dds/tree/src/core/tree/treeTextFormat.ts

```diff
@@ -169,7 +171,9 @@ export function genericTreeDeleteIfEmpty<T>(
 ): void {
 	const children = getGenericTreeFieldMap(node, false);
 	if (Object.prototype.hasOwnProperty.call(children, key)) {
-		if (children[key].length === 0) {
+		// Non null asserting here because of the check above
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		if (children[key]!.length === 0) {
```

Comment:
Can you just do this here?

```suggestion
		if (children[key]?.length === 0) {
```